translations/messages/nl-be.yaml

```diff
@@ -28,6 +29,8 @@ messages:
     warnV1Form: "We hebben gemerkt dat je een {version} formulier aan het editeren bent. Sommige functies zijn gelimiteerd."
     lowestVersionAvailable: "De laagste beschikbare versie: 0"
     highestversionAvailable: "De laatste versie is v{version}"
+    formIsValid: "Je formulier is correct"
+    formIsInvalid: "Je formulier is niet correct. Vul al de velden in."
```

Comment:
I would say: 

Dit formulier is niet volledig. Vul alle velden correct in. 

But maybe leave it for now, I see some more translations that are maybe too literal. We'll take care of them all some time later.

Reply:
Alrighty!

Reply:
Created ticket: https://binnenland.atlassian.net/browse/SFB-183